src/components/patterns/LiveDemo/editor.js

```diff
@@ -86,14 +86,62 @@ if ($err) {
 ?>
 `
 
-const Go = props => `func main() {
+const Go = props => `package main
+
+import (
+  "encoding/json"
+  "net/http"
+  "os"
+)
+
+func main() {
   url := '${apiUrl(props)}'
```

Comment:
This line is still not valid Go, you need to use `"` instead of `'` for strings.

Reply:
good to know, updated 🙂